packages/gasket-plugin-logger/lib/index.js

```diff
@@ -43,6 +43,11 @@ module.exports = {
         gasket.logger = loggers[0];
       }
     },
+    actions(gasket) {
+      return {
+        getLogger: () => gasket.logger
```

Comment:
Why do we need a `getLogger` action? If we have access to the gasket object when calling `gasket.actions.getLogger`, why not use `gasket.logger` straight away?

Reply:
It's just exposed as a utility for now... the actual practicality of it may be up for debate for sure.